demo/core_10_base.js

```diff
@@ -289,16 +320,18 @@ $.user = Object.create($.physical);
 $.user.name = 'User prototype';
 $.user.connection = null;
 $.user.svgText = '<circle cx="50" cy="50" r="10" /><line x1="50" y1="60" x2="50" y2="80"/><line x1="40" y1="70" x2="60" y2="70"/><line x1="50" y1="80" x2="40" y2="100"/><line x1="50" y1="80" x2="60" y2="100"/>';
-$.user.xmlTag = 'user';
 
 $.user.say = function(cmd) {
   if (user.location) {
     // Format:  "Hello.    -or-    say Hello.
     var text = (cmd.cmdstr[0] === '"') ? cmd.cmdstr.substring(1) : cmd.argstr;
-    user.location.tellAll(
-        '<say user="' + $.utils.htmlEscape(user.name) + '" room="' +
-        $.utils.htmlEscape(user.location) + '">' +
-        $.utils.htmlEscape(text) + '</say>');
+    var json = {
```

Comment:
Even though it's arbitrary, I think we should pick a better var name here (and in related places).  We use `cmd` for verb call information, even though it is also plain old JS objects.  How about `activity`, `event` , `update`, `notification`, or even just `msg`?